agents/aws/fence_aws.py

```diff
@@ -122,4 +191,12 @@ def main():
 	sys.exit(result)
 
 if __name__ == "__main__":
+
```

Comment:
I think it'd be better to put this code in top of the main()-function, so it's more logical when looking through the logging code.

Reply:
I tried, but then "logger" won't be a global logger within fence_aws but only to `main()`, and instantiating a new logger at every function call seems like too much, so although the code doesn't look elegant it is the only one I could make it to deliver the results I wanted (as far as logging is concerned).

The main idea is to prevent the usage of the root logger (inherited from "lib/fencing.py") and to have a single-local logger within the fence agent.

Reply:
Ah. In that case I think it'd be better to put it after the imports (so people see that they're global easily).

Reply:
Done! Done! 😄

Reply:
Can you also remove that empty line?

Reply:
> Can you also remove that empty line?

Empty line removed!